.husky/pre-commit

```diff
@@ -15,4 +15,7 @@ if ! git diff-index --quiet HEAD --; then
   echo "Lint fixes applied and staged."
 else
   echo "No lint fixes needed."
-fi
\ No newline at end of file
+fi
+
+# Run tests
+# npm run test
```

Comment:
Un-commenting this line will run tests during pre-commit. I have left this commented out because the tests were quite slow